### ISSUE TITLE
PITY...tons of writing on a single page can be thrown to dustbin. NO RESOURCE CODE!!!! in main window xaml

### ISSUE THREAD

**Issue by cenk100** [MicrosoftDocs/visualstudio-docs#8276]
What a pity.... Two major problems..I don't understand why parts at the core-heart of a sample tutorial by code writing can be this much inattentively and in a shoddy manner be authored.. Because first, the drag drop from data source window to designer window doesn't work.... Second, there are important static resources defined in the database sample, without which WHOLE tutorial deems meaningless... as one can imagine, because the readers are WITHOUT RESOURCE!!!!.... so please , PAY attention , if Microsoft's intention is to HELP people get used to using their products...or if intention is to make people get crazy and stay away from MS, than it works!

And why not give the link to download the FULL CODE?
again copyright?

thnx 
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 42c0fa43-0ca5-9c86-f8c4-4b3eb7176250
* Version Independent ID: bb93e3ff-0d52-9e93-c3a8-545b16f03a30
* Content: [Simple data app with WPF and Entity Framework 6 - Visual Studio (Windows)](https://docs.microsoft.com/en-us/visualstudio/data-tools/create-a-simple-data-application-with-wpf-and-entity-framework-6?view=vs-2022)
* Content Source: [docs/data-tools/create-a-simple-data-application-with-wpf-and-entity-framework-6.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/data-tools/create-a-simple-data-application-with-wpf-and-entity-framework-6.md)
* Product: **visual-studio-windows**
* Technology: **vs-data-tools**
* GitHub Login: @ghogen
* Microsoft Alias: **ghogen**

**Comment by WilliamAntonRohm** [comment]
@cenk100 -- thank you for your feedback.  Please consider these resources:
- [Visual Studio on Microsoft Q&A](https://docs.microsoft.com/answers/products/vs) 
- [Visual Studio MSDN Forum](https://social.msdn.microsoft.com/Forums/vstudio) 
- [Stack Overflow](https://stackoverflow.com/search?q=%22visual+studio%22)  
- [Reddit](https://www.reddit.com/r/VisualStudio)

@ghogen -- please look into this issue.

**Comment by ghogen** [comment]
@cenk100 
What version of Visual Studio were you using? To our knowledge, this tutorial did not work for the earlier releases of Visual Studio 2022, due to known issue, but the problem is fixed in Visual Studio 17.3 Preview 3, so make sure you are using the latest preview of 17.3 if you try this again.  We recently added a note advising users to use that version.

I also added the source code to our tutorial samples repo, https://github.com/MicrosoftDocs/vs-tutorial-samples, and we're publishing an update to the page to add a link.
